deploy-gae-build-task/deploy-gae.ts

```diff
@@ -70,6 +70,11 @@ export async function deployGae({
     }
   }
 
+  // Replace $PROJECTID in imageUrl.
```

Comment:
Nit, this is a nice feature but it is exclusive to the TFS extension, gcloud does not do this. I hesitate as to adding these sort of shortcuts here. The wrapping of gcloud should be really thin.

Reply:
I would rather not remove a working, tested feature. Also, it makes the default value on the UI much closer to the final real value.